hmpps-prisoner-search-indexer/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/listeners/OffenderEventListenerTest.kt

```diff
@@ -75,12 +75,16 @@ internal class OffenderEventListenerTest(@Autowired private val objectMapper: Ob
     @ValueSource(
       strings = [
         "OFFENDER-INSERTED", "OFFENDER-UPDATED", "OFFENDER_DETAILS-CHANGED", "OFFENDER_ALIAS-CHANGED",
+        "KEY_DATE_ADJUSTMENT_UPSERTED", "KEY_DATE_ADJUSTMENT_DELETED",
         "OFFENDER_PHYSICAL_DETAILS-CHANGED", "OFFENDER_IDENTIFIER-UPDATED", "ASSESSMENT-UPDATED_REPUBLISHED",
+        "OFFENDER_CHARGES-UPDATED", "OFFENDER_CHARGES-INSERTED", "OFFENDER_CHARGES-DELETED",
         "OFFENDER_ADDRESS-INSERTED", "OFFENDER_ADDRESS-UPDATED", "OFFENDER_EMAIL-INSERTED", "OFFENDER_EMAIL-DELETED",
         "OFFENDER_EMAIL-UPDATED", "OFFENDER_PHONE-INSERTED", "OFFENDER_PHONE-DELETED", "OFFENDER_PHONE-UPDATED",
         "OFFENDER_ADDRESS_PHONE-INSERTED", "OFFENDER_ADDRESS_PHONE-UPDATED", "OFFENDER_ADDRESS_PHONE-DELETED",
         "OFFENDER_PHYSICAL_ATTRIBUTES-CHANGED", "OFFENDER_IDENTIFYING_MARKS-CHANGED", "OFFENDER_IDENTIFYING_MARKS-DELETED",
         "OFFENDER_ADDRESS-DELETED",
+        "SENTENCE_ADJUSTMENT_UPSERTED", "SENTENCE_ADJUSTMENT_DELETED",
+        "OFFENDER_MARKS_IMAGE-CREATED", "OFFENDER_MARKS_IMAGE-UPDATED", "OFFENDER_MARKS_IMAGE-DELETED",
```

Comment:
Added other missing event types too

Reply:
Do we need some tests for the other event types? Maybe worth putting them in a separate PR? Quite surprised they're missing - why isn't this causing us problems?

Reply:
I wonder why all this KEY_DATE_ADJUSTMENT_ events have been added; I didn't think adjustments was in search,